packages/plugin-figma-blade-coverage/build-figma-plugin.main.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config();
+
+module.exports = function buildFigma(buildOptions) {
+  const define = {};
+
+  // eslint-disable-next-line guard-for-in
+  for (const secretKey in process.env) {
+    let secretValue = process.env[secretKey];
+    if (secretKey === 'SEGMENT_WRITE_KEY') {
+      secretValue = btoa(process.env[secretKey]);
```

Comment:
🤔 why are we doing this btoa?

Reply:
because it'll be passed in headers as Basic Auth. I saw you were passing writekey in body in the code analytics but segment's docs suggest this and it makes sense to pass this in header for security reasons too
<img width="731" alt="Screenshot 2023-07-13 at 6 20 46 PM" src="https://github.com/razorpay/blade/assets/11384858/f48a3c59-e9e8-4da0-8be8-49cfcc046382">